plugins/modules/web_infrastructure/jira.py

```diff
@@ -30,7 +32,7 @@
   operation:
     required: true
     aliases: [ command ]
-    choices: [ create, comment, edit, fetch, transition , link ]
+    choices: [ create, comment, edit, update, fetch, transition , link, search ]
```

Comment:
Alphabetical order of list elements is considered as a good practice (the same for the option in ```argument_spec```)
```suggestion
    choices: [ comment, create, edit, fetch, link, search, transition, update ]
```